### PR TITLE
Initial implementation of Risk SDK

### PR DESCRIPTION
**Pull request by jarrodrobins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,113 @@
...
+Some networking libraries you may be using 
+- [Ktor](https://ktor.io/docs/request.html#parameters)
+- [OkHttp](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-request/-builder/add-header)
+
+```kotlin
+import com.airwallex.risk.AirwallexRisk
+
+val header = AirwallexRisk.header
+
+// Add a header using `header.key` and `header.value` to airwallex.com network requests
```
Comment (richieshilton) [inline README.md]: `header.key` -> `header.field`

File: RiskSdk/src/main/java/com/airwallex/risk/AirwallexRisk.kt [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            if (shared == null) {
+                Log.d(Constants.logTag, Constants.logSdkNotStartedError)
+                return
+            }
+            shared.log(eventType = event, screen = screen)
+        }
+    }
+
+    fun start() {
+        Log.d(Constants.logTag, riskContext.description())
```
Comment (richieshilton) [inline RiskSdk/src/main/java/com/airwallex/risk/AirwallexRisk.kt]: should we remove this for release?

File: README.md (hunk 1 above)
Comment (jarrodrobins) [reply README.md]: Good catch.

File: RiskSdk/src/main/java/com/airwallex/risk/AirwallexRisk.kt (hunk 2 above)
Comment (jarrodrobins) [reply RiskSdk/src/main/java/com/airwallex/risk/AirwallexRisk.kt]: It'll only run in debug mode anyway (the `d` is for debug)

**Review by richieshilton — APPROVED** [review]
👏 Awesome work

**Pull request merged**